src/components/failedTaskBanner/index.tsx

```diff
@@ -53,19 +62,21 @@ const countFailedProcesses = (processes: ProcessWithStatus[]) => {
 };
 
 export default function FailedTaskBanner() {
-    const { runningProcesses } = useContext(RunningProcessesContext);
-    const [failedTasks, setFailedTasks] = useState(countFailedProcesses(runningProcesses));
-    const [data, , fetchData] = useFailedTaskFetcher<ProcessWithStatus>("processes/");
+    const { runningProcesses, completedProcessIds } = useContext(RunningProcessesContext);
+    const [data, , fetchData] = useFailedTaskFetcher<FailedProcess>("processes/");
+    const [failedProcesses, setFailedProcesses] = useState<FailedProcess[]>([]);
+    const [failedTasks, setFailedTasks] = useState(countFailedProcesses([]));
 
     useHttpIntervalFallback(RunningProcessesContext, () => fetchData(0, 10, [], filterFailedTasks));
```

Comment:
Found out the `useFallback` variable is reactive when giving it to the `useHttpIntervalFallback` so will change that, it makes it much more re-usable.